recipes/fast-dds/2.3.X/conanfile.py

```diff
@@ -118,12 +125,7 @@ def requirements(self):
         self.requires("foonathan-memory/0.7.0")
         self.requires("boost/1.73.0")
         if self.options.with_ssl:
-            self.requires("openssl/1.1.1m")
-
-    def export_sources(self):
-        self.copy("CMakeLists.txt")
-        for patch in self.conan_data.get("patches", {}).get(self.version, []):
-            self.copy(patch["patch_file"])
```

Comment:
why it it removed?

Reply:
clicked wrong button .... sorry for that will soon add again